src/app/shared/palette/order-summary/read-order-summary-field.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, ViewChildren, QueryList } from '@angular/core';
+import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+import { ReadOrderSummaryRowComponent } from './read-order-summary-row.component';
+import { FeeValue } from './fee-value.model';
+
+@Component({
+  selector: 'ccd-read-order-summary-field',
+  templateUrl: './read-order-summary-field.html',
+  styleUrls: [
+    './read-order-summary-field.scss'
+  ],
+})
+export class ReadOrderSummaryFieldComponent extends AbstractFieldReadComponent {
+
+  @ViewChildren(ReadOrderSummaryRowComponent)
+  orderSummaryRows: QueryList<ReadOrderSummaryRowComponent>;
```

Comment:
I can't see this used. Is this still needed?

Reply:
yes it is - look at the template

Reply:
I can't see it in the template :)